pyabc/storage/history.py

```diff
@@ -1129,3 +1129,30 @@ def get_population_extended(self, *, m: Union[int, None] = None,
                 df = df_tidy
 
         return df
+
+    @with_session
+    def get_parameter_reference_value(self, refval: str) -> dict:
```

Comment:
I would rather recommend a function that does not take `refval` as an argument, but only self, and then returns the full parameter, e.g. `{'meanX': 3.5, 'meanY': 42,, 'z': 23.513}`